reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+
+  html-uri:
+    tags: ["URIs"]
+    title: "HTML URI"
+    description: "URI to which the HTML representation of this standard is published.".
+
+  pdf-uri:
+    tags: ["URIs"]
+    title: "PDF URI"
+    description: "URI to which the PDF representation of this standard is published.".
+
+  doc-uri:
+    tags: ["URIs"]
+    title: "DOC URI"
+    description: "URI to which the DOC representation of this standard is published.".
+
+  relaton-uri:
+    tags: ["URIs"]
+    title: "Relaton URI"
+    description: "URI to which the Relaton XML representation of this standard is published".
+
+
+  #--- Timestamps ---#
+  copyright-year:
+    tags: ["Timestamps"]
+    title: "Copyrigth year"
+    description: "Year which will be claimed as when the copyright for the document was issued.".
+
+  issued-date:
+    tags: ["Timestamps"]
+    title: "Issued date"
+    description: "Date on which the standard was issued (authorised for publication by the issuing authority).".
+
+  published-date:
+    tags: ["Timestamps"]
+    title: "Published date"
+    description: "Date on which the standard was published (distributed by the publisher).".
+
+  implemented-date:
+    tags: ["Timestamps"]
+    title: "Implemented date"
+    description: "Date on which the standard became active.".
+
+  created-date:
+    tags: ["Timestamps"]
+    title: "created date"
+    description: "Date on which the first version of the standard was created.".
+
+  updated-date:
+    tags: ["Timestamps"]
+    title: "Updated date"
+    description: "Date on which the current version of the standard was updated.".
+
+  obsoleted-date:
+    tags: ["Timestamps"]
+    title: "Obsoleted date"
+    description: "Date on which the standard was obsoleted/revoked.".
+
+  confirmed-date:
+    tags: ["Timestamps"]
+    title: "Confirmed date"
+    description: "Date on which the standard was reviewed and approved by the issuing authority.".
+
+  unchanged-date:
+    tags: ["Timestamps"]
+    title: "Unchanged date"
+    description: "Date on which the standard was last renewed without any changes in content.".
+
+  circulated-date:
+    tags: ["Timestamps"]
+    title: "Circulated date"
+    description: "Date on which the unpublished standard was last circulated officially as a preprint. For standards, this is associated with the latest transition to a formally defined preparation stage, such as Working Draft or Committee Draft.".
+
+  accessed-date:
+    tags: ["Timestamps"]
+    title: "Accessed date"
+    description: "Date on which the standard was last accessed by the compiler of the bibliography; e.g. for a cited online resource, the date on which the document author viewed the resource.".
+
+  date:
+    tags: ["Timestamps"]
+    title: "Date"
+    description: "An arbitrary date in the production of the standard. Content of the attribute should be a token, giving the type of date, then space, then the date itself. Multiple dates can be added as `:date_2:`, `:date_3:`, etc.".
```

Comment:
Have not identified such an issue: if we know of a type of date we haven't covered, we just add it to the relaton spec. This is intended as a fallback, in case a date is genuinely underspecified, which can happen in mapping from other bibliographis standards. (STS has such an underspecified date. Or rather, the specification of that date is optional and free-text.)